configs/bart2vm16gpu.json

```diff
@@ -15,5 +15,6 @@
   "model_display_name" : "bart",
   "deploy_machine_type" : "n1-standard-32",
   "deploy_gpu_type" : "NVIDIA_TESLA_V100",
-  "deploy_gpu_count" : 2
+  "deploy_gpu_count" : 2,
+  "pipeline_node_memory_limit": "16G"
```

Comment:
Why do we set it here even though the default is 16G?

Reply:
Just realized that there is a way to set default parameters within the pipeline function itself. Will change the default value to be set there instead of in this config file.

Reply:
For more context, the KFP itself breaks if you reference a param that isn't present in the config.

I had expected it to just default to None.

Reply:
Moved default to the pipeline function.